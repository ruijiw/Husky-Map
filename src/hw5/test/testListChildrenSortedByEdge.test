# Specification testing file for star war
# Tests that sort algorithm is correct

# Create the graph
CreateGraph h

# Add multiple nodes
AddNode h n
AddNode h n1
AddNode h n2
AddNode h n3

# Add an edge
AddEdge h n n1 eZ
AddEdge h n n1 eA
AddEdge h n n3 e3
AddEdge h n n2 eA
AddEdge h n n2 eJ
AddEdge h n n2 eZ

# Print the nodes in the graph and the outgoing edges from n
ListChildren h n