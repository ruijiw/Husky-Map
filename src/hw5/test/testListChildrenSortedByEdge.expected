# Specification testing file for star war
# Tests that sort algorithm is correct

# Create the graph
created graph h

# Add multiple nodes
added node n to h
added node n1 to h
added node n2 to h
added node n3 to h

# Add an edge
added edge eZ from n to n1 in h
added edge eA from n to n1 in h
added edge e3 from n to n3 in h
added edge eA from n to n2 in h
added edge eJ from n to n2 in h
added edge eZ from n to n2 in h

# Print the nodes in the graph and the outgoing edges from n
the children of n in h are: n1(eA) n1(eZ) n2(eA) n2(eJ) n2(eZ) n3(e3)